WebApiBackend/WebApiBackend/Controllers/FlatController.cs

```diff
@@ -48,5 +48,49 @@ public ActionResult<List<DisplayMemberDTO>> GetMembers()
             return _MemberMapper.Map<List<DisplayMemberDTO>>(members);
         }
 
+        [HttpGet("AddUserToFlat/{FlatId}/{userId}")]
+        [Authorize]
+        public ActionResult<AddUserToFlatDto> AddUserToFlat(int FlatId, int userId)
+        {
+
+            User finduser = _context.User.Find(userId);
```

Comment:
Neither the UserID or the FlatID are stored by the frontend, and so it is not possible for them to make a Get Request containing the flatID and user ID. Please look at the previous method in the flat controller.

The proper way to obtain the ID of a user making the request is shown below.

```
var identity = (ClaimsIdentity)HttpContext.User.Identity;
 int userID = Int16.Parse(identity.FindFirst(ClaimTypes.NameIdentifier).Value);
```